CentralityiOS/CentralityiOS/ViewControllers/AlertsModalViewController.m

```diff
@@ -0,0 +1,174 @@
+//
+//  AlertsModalViewController.m
+//  CentralityiOS
+//
+//  Created by Samuel Osa-Agbontaen on 8/1/22.
+//
+
+#import "AlertsModalViewController.h"
+#import "ReceiverCell.h"
+#import "CentralityHelpers.h"
+
+@interface AlertsModalViewController ()<UITableViewDelegate, UITableViewDataSource>
+
+@end
+
+@implementation AlertsModalViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    self.receiverTableView.dataSource = self;
+    self.receiverTableView.delegate = self;
+    [self fetchReceivedTasks];
+}
+- (IBAction)closeAction:(id)sender {
+    [self dismissViewControllerAnimated:YES completion:^{}];
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section{
+    return self.arrayOfPendingSharedTasks.count;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    TaskObject *task = self.arrayOfPendingSharedTasks[indexPath.row];
+    [task.owner fetchIfNeeded];
+    ReceiverCell *cell = [tableView dequeueReusableCellWithIdentifier:@"ReceiverCell" forIndexPath:indexPath];
+    cell.taskNameLabel.text = task.taskTitle;
+    cell.taskDescLabel.text = task.taskDesc;
+    cell.taskOwnerLabel.text = [NSString stringWithFormat:@"Owned by : %@", task.owner.username];
+    
+    NSString* allUsers = @"";
+    NSString* displayMessage = [[NSString alloc] init];
+    if (task.acceptedUsers.count > 0){
+        for (NSInteger index = 0; index < task.sharedOwners.count; index++){
+            allUsers = [allUsers stringByAppendingString:[task.acceptedUsers[index] fetchIfNeeded].username];
```

Comment:
if `allUsers` is a `NSMutableString` instead of a `NSString` you can do `[allUsers appendString:username]` instead. I believe this is slightly more efficient.